src/main/resources/assets/unlimitedchiselworks/ucwdefs/chisel/iceandfire.json

```diff
@@ -0,0 +1,95 @@
+{
+  "modid": ["chisel", "iceandfire"],
+  "blocks": [
+	{
+      "from": {
+        "block": "iceandfire:chared_cobblestone"
+      },
+      "mode": "blend",
+      "through": {
+        "block": "chisel:cobblestone",
+        "iterate": [
+          "variation"
+        ]
+      },
+      "based_upon": {
+        "state": "minecraft:cobblestone"
+      }
+    },
+	{
+      "from": {
+        "block": "iceandfire:chared_cobblestone"
+      },
+      "mode": "blend",
+      "through": {
+        "block": "chisel:cobblestone1",
+        "iterate": [
+          "variation"
+        ]
+      },
+      "based_upon": {
+        "state": "minecraft:cobblestone"
+      }
+    },
+	{
+      "from": {
+        "block": "iceandfire:chared_cobblestone"
+      },
+      "mode": "blend",
+      "through": {
+        "block": "chisel:cobblestone2",
+        "iterate": [
+          "variation"
+        ]
+      },
+      "based_upon": {
+        "state": "minecraft:cobblestone"
+      }
+    },
+	{
+      "from": {
+        "block": "iceandfire:chared_stone"

```

Comment:
Is that a stone brick, or just a stone? Gotta be a stone brick.

Reply:
It's just plain stone, there are no bricks. It seemed to work okay, though.